RileyLinkKit/Extensions/NSDate.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  NSDate.swift
+//  RileyLink
+//
+//  Created by Pete Schwamb on 5/18/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+public func ==(lhs: NSDate, rhs: NSDate) -> Bool {
```

Comment:
ah, right, because the protocol is public. I'm concerned that two frameworks with the same public extension would conflict...